packages/x-codemod/README.md

```diff
@@ -95,31 +97,31 @@ The list includes these transformers
 Renames the Tree View component to Simple Tree View
 
 ```diff
--import { TreeView } from '@mui/x-tree-view';
-+import { SimpleTreeView } from '@mui/x-tree-view';
+- import { TreeView } from '@mui/x-tree-view';
```

Comment:
Olivier has recently fixed such formatting issues. 🙈 
https://github.com/mui/mui-x/pull/15575#discussion_r1855318679

Reply:
Ah ok, so it is the other way around :)
I'll update this